wasm/tx_template/Cargo.toml

```diff
@@ -25,6 +25,16 @@ borsh-derive = {git = "https://github.com/heliaxdev/borsh-rs.git", rev = "cd5223
 borsh-derive-internal = {git = "https://github.com/heliaxdev/borsh-rs.git", rev = "cd5223e5103c4f139e0c54cf8259b7ec5ec4073a"}
 borsh-schema-derive-internal = {git = "https://github.com/heliaxdev/borsh-rs.git", rev = "cd5223e5103c4f139e0c54cf8259b7ec5ec4073a"}
 
+# patched to a commit on the `eth-bridge-integration` branch of our fork
+tendermint = {git = "https://github.com/heliaxdev/tendermint-rs.git", rev = "bc0b6ac47b3bfc1ee8b944341b654b867b3b5d0b"}
+tendermint-proto = {git = "https://github.com/heliaxdev/tendermint-rs.git", rev = "bc0b6ac47b3bfc1ee8b944341b654b867b3b5d0b"}
+tendermint-testgen = {git = "https://github.com/heliaxdev/tendermint-rs.git", rev = "bc0b6ac47b3bfc1ee8b944341b654b867b3b5d0b"}
+tendermint-light-client-verifier = {git = "https://github.com/heliaxdev/tendermint-rs.git", rev = "bc0b6ac47b3bfc1ee8b944341b654b867b3b5d0b"}
+
+# patched to a commit on the `eth-bridge-integration` branch of our fork
+ibc = {git = "https://github.com/heliaxdev/ibc-rs.git", rev = "451ff75591d9de48b7147a651858c3daeb5e2436"}
+ibc-proto = {git = "https://github.com/heliaxdev/ibc-rs.git", rev = "451ff75591d9de48b7147a651858c3daeb5e2436"}
+
```

Comment:
We didn't have to patch these things in our wasm crates before, as the namada crates were pulling in the forks for us indirectly. Now, because the namada crates are patched at the workspace level, we have to also patch anywhere we use the individual namada crates.

In some sense, it would make sense for individual namada crates to pull in the forks directly rather than via workspace-level `Cargo.toml` patching, but it's not possible to patch individual crates in a workspace - also, we are then unable to patch tendermint-rs and the like whenever we are using those crates outside of the workspace due to https://github.com/rust-lang/cargo/issues/5478 :(